lib/internal/Magento/Framework/Image/Adapter/ImageMagick.php

```diff
@@ -35,6 +36,24 @@ class ImageMagick extends AbstractAdapter
         'sharpen' => ['radius' => 4, 'deviation' => 1],
     ];
 
+    /**
+     * @var \Imagick
+     */
+    protected $_imageHandler;
```

Comment:
Why you need this var? It should be inherited from the parent class

Reply:
It provides type information for within the ImageMagick class.